Index: TIL/인프런/섹션9
===================================================================
diff --git a/TIL/인프런/섹션9 b/TIL/인프런/섹션9
deleted file mode 100644
--- a/TIL/인프런/섹션9	(revision 1bc683e2c06705133385dfd767598399b5b6ebb3)
+++ /dev/null	(revision 1bc683e2c06705133385dfd767598399b5b6ebb3)
@@ -1,1 +0,0 @@
-ㅈㄷㄹ
\ No newline at end of file
diff --git a/TIL/인프런/섹션9.md b/TIL/인프런/섹션9.md
new file mode 100644
